Installing packages on nodejs v14 fails while building nsfw gyp

## Summary

Installing packages on nodejs v14.3.0 fails while building [`nsfw`] package.

```shell
~/w/ark-core!$ nodenv global 14.3.0
~/w/ark-core $ rm -rf node_modules/; and yarn; and yarn build
yarn install v1.22.4
[1/4] Resolving packages...
[2/4] Fetching packages...
...
make: *** [nsfw.target.mk:128: Release/obj.target/nsfw/src/NSFW.o] Error 1
make: Leaving directory '/home/rainydio/work/ark-core/node_modules/nsfw/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
```

https://gist.github.com/rainydio/637934ae90a1fa636dcc1067ad3f7eab

Installing latest version of `nsfw` into new nodejs project succeeds.

[`nsfw`]: https://www.npmjs.com/package/nsfw

--- a/packages/core-kernel/package.json
+++ b/packages/core-kernel/package.json
@@ -34,7 +34,7 @@
         "inversify": "^5.0.1",
         "log-process-errors": "^5.0.3",
         "nanomatch": "^1.2.13",
-        "nsfw": "^1.2.5",
+        "nsfw": "^2.0.0",
         "reflect-metadata": "^0.1.13",
         "semver": "^6.3.0",
         "type-fest": "^0.14.0"

--- a/packages/core-kernel/src/services/config/watcher.ts
+++ b/packages/core-kernel/src/services/config/watcher.ts
@@ -1,30 +1,7 @@
-import nsfw from "nsfw";
+import nsfw, { ActionType, NSFW } from "nsfw";
 
 import { Application } from "../../contracts/kernel";
 import { Identifiers, inject, injectable } from "../../ioc";
-
-/**
- * @interface FileEvent
- */
-interface FileEvent {
-    /**
-     * @type {number}
-     * @memberof FileEvent
-     */
-    action: number;
-
-    /**
-     * @type {string}
-     * @memberof FileEvent
-     */
-    directory: string;
-
-    /**
-     * @type {string}
-     * @memberof FileEvent
-     */
-    file: string;
-}
 
 /**
  * @export
@@ -44,10 +21,10 @@ export class Watcher {
 
     /**
      * @private
-     * @type {nsfw}
+     * @type {NSFW}
      * @memberof Watcher
      */
-    private watcher: nsfw;
+    private watcher!: NSFW;
 
     /**
      * @returns {Promise<void>}
@@ -56,9 +33,9 @@ export class Watcher {
     public async boot(): Promise<void> {
         const configFiles: string[] = [".env", "delegates.json", "peers.json", "plugins.js", "plugins.json"];
 
-        this.watcher = await nsfw(this.app.configPath(), (events: FileEvent[]) => {
+        this.watcher = await nsfw(this.app.configPath(), (events) => {
             for (const event of events) {
-                if (configFiles.includes(event.file) && event.action === nsfw.actions.MODIFIED) {
+                if (event.action === ActionType.MODIFIED && configFiles.includes(event.file)) {
                     this.app.reboot();
                 }
             }

